tbx/core/blocks.py

```diff
@@ -15,11 +15,15 @@
 from wagtail.embeds.blocks import EmbedBlock as WagtailEmbedBlock
 from wagtail.embeds.embeds import get_embed
 from wagtail.embeds.exceptions import EmbedException
+from wagtail.images import get_image_model
 from wagtail.images.blocks import ImageChooserBlock
+from wagtail.models import Page
 from wagtail.snippets.blocks import SnippetChooserBlock
 from wagtailmarkdown.blocks import MarkdownBlock
 from wagtailmedia.blocks import VideoChooserBlock
 
+Image = get_image_model()
```

Comment:
I did discuss this with Andy, but I'm still not completely clear why we don't use the `CustomImage` model here as in the model files.

Reply:
I put that under force of habit